3p/bodymovinanimation.js

```diff
@@ -29,10 +30,14 @@ let animationHandler;
 
 /**
  * @param {!Window} global
+ * @param {string} renderer
  * @param {!Function} cb
  */
-function getBodymovinAnimationSdk(global, cb) {
-  loadScript(global, 'https://cdnjs.cloudflare.com/ajax/libs/bodymovin/4.13.0/bodymovin_light.min.js', function() {
+function getBodymovinAnimationSdk(global, renderer, cb) {
+  const scriptToLoad = renderer === 'svg' ?
+    'https://cdnjs.cloudflare.com/ajax/libs/bodymovin/4.13.0/bodymovin_light.min.js' :
```

Comment:
`preconnectCallback` in the component file would also need to change to pick the right js based on renderer.